panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
Oh, there's a misunderstanding here, that is *my fault*. Now I get what you meant when you asked, "Should the initial panel showup be animated or not?" Yes, it seems to me natural that animated panels to be shown with animation at start-up. With your method, all auto-hiding panels would be suddenly shown at start-up, whether they have animation or not.

Anyway, I was working with my older computer when I tested your suggestion and saw a "flash" at start-up with *my* method. I'll try to fix it but if not successful, I'll have no choice other than using your method (no animation at start-up).